### ISSUE TITLE
'clr-oci-runtime ps CID' shows empty

### ISSUE THREAD

**Issue by sameo** [clearcontainers/runtime#13]
_From @devimc on July 18, 2016 18:53_

ps command has not implementation

_Copied from original issue: 01org/cc-oci-runtime#24_

**Comment by sameo** [comment]
_From @grahamwhaley on July 29, 2016 11:57_

Is this the ps that will be used by Docker/containerd ?

**Comment by sameo** [comment]
_From @devimc on August 4, 2016 13:39_

@grahamwhaley  yes , this bug depends on #18

**Comment by grahamwhaley** [comment]
@sboeuf @devimc I think this is a duplicate of #95 , in which case, let's close this one - you agree?

**Comment by sboeuf** [comment]
Yes I agree, I close this one.
